### ISSUE TITLE
Feature Request! - Recompile All (If upgrading from older FNR generated code can make issues)

### ISSUE THREAD

**Issue by Fashdey** [BeardedManStudios/ForgeNetworkingRemastered#43]
Tl;dr - Make a Recompile all and it should work flawlessly!

### Version Number and Operating System(s):
All is related to DevBranch
(Old)Build Date: 11/07-17
(New)Buld Date: 31/07-17
Unity 2017

### Expected behavior:
No issues in running the game with network capabilities, after FNR Upgrade

### Actual behavior:
Clients may get a wrong cast error on Generated code

```InvalidCastException: Specified cast is not valid.
  at BeardedManStudios.Forge.Networking.Generated.GameManagerBehavior.Initialize (BeardedManStudios.Forge.Networking.NetworkObject obj) [0x0002e] in D:\Unity Projects\ProjectCTF\Assets\Bearded Man Studios Inc\Generated\UserGenerated\GameManagerBehavior.cs:21 
  at BeardedManStudios.Forge.Networking.Unity.NetworkManager.SceneReady (UnityEngine.SceneManagement.Scene scene, UnityEngine.SceneManagement.LoadSceneMode mode) [0x0024d] in D:\Unity Projects\ProjectCTF\Assets\Bearded Man Studios Inc\Scripts\NetworkManager.cs:595 
  at BeardedManStudios.Forge.Networking.Unity.NetworkManager.OnLevelFinishedLoading (UnityEngine.SceneManagement.Scene scene, UnityEngine.SceneManagement.LoadSceneMode mode) [0x00004] in D:\Unity Projects\ProjectCTF\Assets\Bearded Man Studios Inc\Scripts\NetworkManager.cs:363 
  at UnityEngine.SceneManagement.SceneManager.Internal_SceneLoaded (UnityEngine.SceneManagement.Scene scene, UnityEngine.SceneManagement.LoadSceneMode mode) [0x00013] in C:\buildslave\unity\build\artifacts\generated\common\runtime\SceneManagerBindings.gen.cs:245 
```

Mentioned line is in the generated code which is
**networkObject = (GameManagerNetworkObject)obj;**

### Steps to reproduce:
Using a build of (Old) and a build of (New) should reproduce this
Make the game and add some instructions, which should be able to recall this

(I'll keep an old version of our game hanging around if there is research on this to be made)
- Just ask for it

#### [Optional] Discord Username:
Fashdey#5329

### How to fix
Went to the FNR Wizard and just clicked the Save & Recompile and it worked flawlessly!
